### ISSUE TITLE
[Question]: Does IBrowserContext.Pages preserve order?

### ISSUE THREAD

**Issue by RenderMichael** [microsoft/playwright-dotnet#2358]
### Your question

Does calling
``` csharp
var pages = Context.Pages;
```
Follow the left-to-right order the same way using a browser manually has? If I index into a specific tab, will it be the right one?

**Comment by mxschmitt** [comment]
Yes it should do! Popups will land at the end. But Context.Pages don't only contain tabs, they also contain Popups, since they belong to the same browser context, so be aware.
